app/include/zmk/hid.h

```diff
@@ -169,17 +169,26 @@ struct zmk_hid_consumer_report {
 zmk_mod_flags_t zmk_hid_get_explicit_mods();
 int zmk_hid_register_mod(zmk_mod_t modifier);
 int zmk_hid_unregister_mod(zmk_mod_t modifier);
+bool zmk_hid_mod_is_pressed(zmk_mod_t modifier);
+
 int zmk_hid_register_mods(zmk_mod_flags_t explicit_modifiers);
 int zmk_hid_unregister_mods(zmk_mod_flags_t explicit_modifiers);
 int zmk_hid_implicit_modifiers_press(zmk_mod_flags_t implicit_modifiers);
 int zmk_hid_implicit_modifiers_release();
+
 int zmk_hid_keyboard_press(zmk_key_t key);
 int zmk_hid_keyboard_release(zmk_key_t key);
 void zmk_hid_keyboard_clear();
+bool zmk_hid_keyboard_is_pressed();
```

Comment:
Messed up declaration?

```suggestion
bool zmk_hid_keyboard_is_pressed(zmk_key_t key);
```